templates/README.md

```diff
@@ -40,12 +40,15 @@ For more information about the terraform in this repo, please see [Link][zero-tf
 
 Before using the cluster the first time you'll need to set up your local `kubectl` context:
 ```shell
-make update-k8s-conf
+$ make update-k8s-conf
+...
+Updated context <context name> in ~/.kube/config
```

Comment:
`Updated` or `Update` ?

Reply:
That's meant to show the output of the `make update-k8s-conf` command (doesn't start with `$`)